src/constants.py

```diff
@@ -21,16 +21,17 @@
 TLS_SSL_KEY_FILE = "custom-server-key.pem"
 TLS_SSL_CERT_FILE = "custom-server-cert.pem"
 CHARMED_MYSQL_SNAP_NAME = "charmed-mysql"
-CHARMED_MYSQL_SNAP_CHANNEL = "8.0/edge"
+CHARMED_MYSQL_SNAP_CHANNEL = "8.0/edge/snap-dir"
```

Comment:
i will update this to `8.0/edge` as soon as [PR 11 in charmed mysql snap](https://github.com/canonical/charmed-mysql-snap/pull/11/files) is merged

Reply:
updated in [b0dd8e6](https://github.com/canonical/mysql-operator/pull/158/commits/b0dd8e664ad193d683415fa5d3f5e77eeab378ff)